core/store/src/config.rs

```diff
@@ -35,7 +43,7 @@ pub struct StoreConfig {
     /// Increasing DBCol::State cache size helps making storage more efficient. On the other hand we
     /// don't want to increase hugely requirements for running a node so currently we use a small
     /// default value for it.
-    pub col_state_cache_size: bytesize::ByteSize,
+    pub col_state_block_cache_size: bytesize::ByteSize,
```

Comment:
In my understanding config is stored as json, so this renaming will break existing configuration

Reply:
In my understanding this will only make the config parameter invalid, it will be read but ignored, and the user only gets this error back: `WARN neard: /home/ubuntu/.near/config.json: encountered unrecognised field: store.col_state_cache_size`. I am not sure what you mean by breaking the existing configuration.

Reply:
The scope of this change is FlatState column. So I think it is better to avoid renaming config params as it does have side effects on State column.

Reply:
Aha ok makes sense